src/declaration.d

```diff
@@ -2096,7 +2106,12 @@ extern (C++) class VarDeclaration : Declaration
         if (tv.ty == Tstruct)
         {
             StructDeclaration sd = (cast(TypeStruct)tv).sym;
-            if (!sd.dtor || !type.size())
+            if (!sd.dtor)
+                return null;
+
+            const sz = type.size();
+            assert(sz != SIZE_INVALID);
```

Comment:
Apparently the `sz` can have an invalid size: https://issues.dlang.org/show_bug.cgi?id=17423